corehq/apps/api/resources/__init__.py

```diff
@@ -93,7 +93,7 @@ def dispatch(self, request_type, request, **kwargs):
         if request.user.is_superuser or domain_has_privilege(request.domain, privileges.API_ACCESS):
             return super(HqBaseResource, self).dispatch(request_type, request, **kwargs)
         else:
-            raise ImmediateHttpResponse(HttpResponse(json.dumps({"error": "not authorized"}),
+            raise ImmediateHttpResponse(HttpResponse(json.dumps({"error": "You have lost access to this feature"}),
```

Comment:
Can we just change the copy to "Your current plan does not have access to this feature"?